dev/make-distribution.sh

```diff
@@ -161,7 +161,7 @@ fi
 # Build uber fat JAR
 cd "$SPARK_HOME"
 
-export MAVEN_OPTS="${MAVEN_OPTS:--Xss128m -Xmx4g -Xmx4g -XX:MaxMetaspaceSize=2g -XX:ReservedCodeCacheSize=128m}"
```

Comment:
It was added by https://github.com/apache/spark/pull/37510. Seems it was added also because `make-distributions.sh` failed to build.

Reply:
I see, but there still seems to be build fail.

Reply:
Yea, I can reproduce it locally too. I think there are some changes since #37510.

Reply:
Thank @viirya